internal/querycoordv2/dist/dist_handler.go

```diff
@@ -135,7 +136,7 @@ func (dh *distHandler) updateSegmentsDistribution(resp *querypb.GetDataDistribut
 func (dh *distHandler) updateChannelsDistribution(resp *querypb.GetDataDistributionResponse) {
 	updates := make([]*meta.DmChannel, 0, len(resp.GetChannels()))
 	for _, ch := range resp.GetChannels() {
-		channelInfo := dh.target.GetDmChannel(ch.GetChannel())
+		channelInfo := dh.target.Current.GetDmChannel(ch.GetChannel())
```

Comment:
It's better to export a public function in target manager rather than `Current` target